lib/Test/Nginx/Util.pm

```diff
@@ -1893,6 +1904,12 @@ start_nginx:
             write_user_files($block, $rand_ports);
             write_config_file($block, $config, $rand_ports);
             #warn "nginx binary: $NginxBinary";
+
+            #before_start
+            for my $hdl (@BeforeStartHandlers) {
```

Comment:
Perhaps we can move this statement to after the $NginxVersion assignment is complete, i.e. line 1924